### DISCUSSION TITLE
Uptime is shuttime?

### DISCUSSION THREAD

**Discussion by lollita** [winsiderss/systeminformer#1728] [Questions / Answers] [answered]
In CPU information, the uptime non display the time from boot as windows task manager but the time from last shutdown. Do it is a bug?
It would be useful to have both information.
Also the utilitation is every half compared to WTM.
[See here.](https://interno.altervista.org/temp/uptime.jpg)

**Comment by dmex** [comment, marked as answer]
> the utilitation is every half compared

Task Manager reports `Processor Utility` which is not the same as metric as utilization. To compare the result from Task Manager you need to multiply the percentage by the processor frequency (utility * core freq / current freq * 100).

Based on your screenshot Task Manager's 18% * 2GHz / 3GHz  * 100 == 12% is almost the same as System Informer's 10% but still not as accurate. Task Manager hasn't been reporting accurate utilization for a long time and there are thousands of complaints and news articles about it: https://aaron-margosis.medium.com/task-managers-cpu-numbers-are-all-but-meaningless-2d165b421e43

> the uptime non display the time from boot as windows task manager but the time from last shutdown. 

Uptime is the amount of time since last shutdown. Task Manager is showing 3 hours and System Informer is showing 21 hours.

Open Powershell and Execute `Get-Uptime` and System Informer should be showing the same value:

![image](https://github.com/winsiderss/systeminformer/assets/1306177/acf7dd1b-5cf4-4588-bffa-e45ddf289c32)
